docs/source/testing.rst

```diff
@@ -26,27 +26,37 @@ Unit testing using OpenShift
 
 The ``Makefile`` provides convenience commands for enabling port-forwarding. This is required for accessing a database pod hosted within an OpenShift environment.
 
-Example:
-::
+Example::
+
     $ make oc-create-all
     $ make oc-forward-ports
     $ tox
     $ make stop-forwarding-ports
 
-Smoke testing
-=============
-
-Running the QE functional and smoke tests is achieved through the scripts provided in ``koku.git/testing``
+Unit testing log messages
+-------------------------
+The logger is disabled by default during unit tests. If you are building a unit test that asserts log messages, you must re-enable the logger. For example::
 
-Requirements
-------------
-In order to run the test suite, there need to be clones of the following git repositories:
+    import logging
+    with self.assertLogs(logger='masu.external.downloader.aws.aws_report_downloader', 
+    level='WARN') as cm:
+                logging.disable(logging.NOTSET)
+                self.aws_report_downloader._remove_manifest_file("None")
+                self.assertEqual(['WARN: Could not delete manifest file at'], cm.output)
 
-  - koku
-  - iqe
-  - hccm-plugin
+If you observe the following error in the tox tests, you may sometimes ignore it, due to the environment variable DEBUG=TRUE sometimes not being TRUE::
```

Comment:
Let's avoid the term "tox tests" and use "unit tests" instead.

Tox is just the tool we use to run the unit tests. The phrase "unit tests" appears to be used more frequently is this doc, so let's standardize on that for consistency.

```suggestion
If you observe the following error when running the unit tests, it may be a false error. This can be due to some tests expecting the DEBUG setting to be TRUE::
```

Though, really, this is something I'd consider a bug that needs to be fixed. Tests shouldn't fail based on the value of DEBUG. They should either be skipped or succeed. I suggest opening a bug about this.